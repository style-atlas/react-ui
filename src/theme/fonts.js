export const body = ["Open Sans", "sans-serif"];
export const header = ["Roboto Slab", "serif"];
