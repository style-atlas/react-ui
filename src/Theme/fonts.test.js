import React from "react";
import * as fonts from "./fonts";

test("Fonts should be as snapshot defines", () => {
  expect(fonts).toMatchSnapshot();
});
