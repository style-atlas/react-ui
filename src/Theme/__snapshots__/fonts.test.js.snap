// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Fonts should be as snapshot defines 1`] = `
Object {
  "body": Array [
    "Open Sans",
    "sans-serif",
  ],
  "header": Array [
    "Roboto Slab",
    "serif",
  ],
}
`;
